_data/sites.json

```
@@ -5388,6 +5388,17 @@
             "support.docusign.com"
         ]
     },
+    {
+        "name": "Doksure",
+        "url": "https://doksure.com/supprimer-son-compte",
+        "notes": "GDPR compliant. Go to Settings > Account > Delete Account.",
+        "notes_fr": "Conforme RGPD. Allez dans Paramètres > Compte > Supprimer son compte."
+        "difficulty": "easy",
+        "domains": [
+          "doksure.com"
+        ],
```

```suggestion
        "domains": ["doksure.com"],
```
We're making domains a one-liner if it is a single domain